src/h5web/visualizer/Visualizer.tsx

```diff
@@ -1,22 +1,28 @@
-import { ReactElement, Suspense, useState } from 'react';
-import type { Entity } from '../providers/models';
+import { ReactElement, Suspense, useContext, useState } from 'react';
 import styles from './Visualizer.module.css';
-import { getSupportedVis } from './utils';
+import { getDefaultEntity, getSupportedVis } from './utils';
 import { VIS_DEFS, Vis } from '../visualizations';
 import VisSelector from './VisSelector';
 import Loader from './Loader';
 import Profiler from '../Profiler';
 import ErrorMessage from './ErrorMessage';
 import { ErrorBoundary } from 'react-error-boundary';
+import { ProviderContext } from '../providers/context';
 
 interface Props {
-  entity?: Entity;
+  path: string;
 }
 
 function Visualizer(props: Props): ReactElement {
-  const { entity } = props;
+  const { path } = props;
 
-  const { supportedVis, error } = getSupportedVis(entity);
+  const { entitiesStore } = useContext(ProviderContext);
+  const entity = entitiesStore.get(path);
+
+  // Resolve any `default` attribute(s) to find entity to visualize
+  const defaultEntity = getDefaultEntity(entity, entitiesStore);
```

Comment:
The line/function that does all the default path resolution work. After this we don't worry about default paths ever again! 😄

Reply:
lol 😁